nxdrive/data/i18n/i18n-fr.json

```
@@ -239,8 +239,8 @@
     "NETWORK_ERROR_UnknownNetworkError": "Erreur inconnue",
     "NETWORK_ERROR_UnknownProxyError": "Erreur inconnue",
     "NEW_ENGINE": "Ajouter un compte",
-    "NEW_REMOTE_FOLDER": "New remote folder",
-    "NEW_REMOTE_FOLDER_TOOLTIP": "Create a new target remote folder in the selected path where all files from the session will be uploaded to.",
+    "NEW_REMOTE_FOLDER": "Nouveau dossier distant",
+    "NEW_REMOTE_FOLDER_TOOLTIP": "Créer un nouveau dossier distant dans le chemin sélectionné où tous les fichiers de la session seront téléchargés.",
```

```suggestion
    "NEW_REMOTE_FOLDER_TOOLTIP": "Créer un nouveau dossier sur le serveur au niveau du chemin sélectionné où tous les fichiers de la session seront envoyés/téléversés.",
```

"chemin" does not feel correct/good 🤔